mounts/org.osbuild.ostree.deployment

```diff
@@ -30,7 +30,11 @@ SCHEMA_2 = """
   "type": { "type": "string" },
   "options": {
     "type": "object",
-    "required": ["deployment"],
+    "oneOf": [{
+      "required": ["deployment"]
+    }, {
+      "required": ["default-deployment"]
+    }],
```

Comment:
A modification on `mounts/org.osbuild.ostree.deployment` was discovered after realizing that the deployment option for the `org.osbuild.fstab` stage is specified under a mount. In order for the schema to work out, a modification needed to be made in this file. However, with this change any stage that also specifies an ostree deployment as a mount should be able to utilze the `default-deployment` option.